src/views/PancakeSquad/assets/EventStepsBottomWave.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import { Svg, SvgProps } from '@pancakeswap/uikit'
+
+const Icon: React.FC<SvgProps & { isDark: boolean }> = ({ isDark, ...props }) => {
+  return (
+    <Svg preserveAspectRatio="none" viewBox="0 0 1280 17" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+      <path
+        d="M0 16.002V0.00195312C159.43 9.20767 387.163 14.9682 640 14.9682C892.837 14.9682 1120.57 9.20767 1280 0.00195312V16.002H0Z"
+        fill="url(#paint0_linear)"
+      />
+      <defs>
+        <linearGradient id="paint0_linear" gradientUnits="userSpaceOnUse">
```

Comment:
Just a note - its better to give more descriptive name to svg ids. We had a problem where some ids were had the same name and were interfering with each other. E.g. if some other `<linearGradient>` on the page would use `paint0_linear` id but with different colors - this might use that other `pain0_linear` instead of the one here.